env0/resource_environment.go

```diff
@@ -109,6 +110,12 @@ func resourceEnvironment() *schema.Resource {
 				Description: "destroy safegurad",
 				Optional:    true,
 			},
+			"wait": {
```

Comment:
This wait feature could be more configurable in the future. For example, right now it's still waiting if deployment is `WAITING_FOR_USER`, but some user might want a different behaviour

Let's keep this new feature open for extension.

Maybe something along the lines of `waitFor: 'ACK' | 'FULLY_DEPLOYED'`, with `ACK` being the default